doc/Makefile.autosetup

```diff
@@ -1,7 +1,7 @@
-MAKEDOC_CPP = $(CC_FOR_BUILD) $(CPPFLAGS) -D_MAKEDOC -E -C -I.
+MAKEDOC_CPP = $(CC_FOR_BUILD) -D_MAKEDOC -E -C -I.
 
 doc/makedoc$(EXEEXT): doc/makedoc.c
-	$(CC_FOR_BUILD) $(CPPFLAGS) $(CFLAGS) -o $@ doc/makedoc.c
+	$(CC_FOR_BUILD) -std=c99 -I. -o $@ doc/makedoc.c
```

Comment:
This gives me build warnings, now.
Adding `#include <strings.h>` to `doc/makedoc.c` fixes one.

And, this comes from not having `_GNU_SOURCE` defined (which is in CFLAGS).
```
doc/makedoc.c: In function ‘main’:
doc/makedoc.c:1418:15: warning: implicit declaration of function ‘getopt’; [-Wimplicit-function-declaration]
   while ((c = getopt(argc, argv, "cmsd")) != EOF)
```